src/config_file.c

```diff
@@ -106,6 +106,12 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 			goto free;
 		}
 		g_key_file_remove_key(key_file, "system", "efi-use-bootnext", NULL);
+	} else if (g_strcmp0(c->system_bootloader, "custom") == 0) {
+		c->custom_script = key_file_consume_string(key_file, "system", "bootloader-custom-script", NULL);
+		if (!c->custom_script) {
```

Comment:
A bit of bikeshedding, but the custom bootloader interface need not be a script. Perhaps just `bootloader-interface` or `bootloader-handler`. And another bit of bikeshedding, perhaps the setting even belongs in the `[handlers]` section, the documentation for that does read "Handlers allow to customize RAUC by placing scripts in the system that RAUC can call for different purposes.".

Reply:
s/bootloader-custom-script/bootloader-custom-backend/
and
s/custom_script/bootloader_custom_backend/

Moved to handlers. Thanks